src/main/java/com/singh/astha/notificationservice/enums/DeviceType.java

```diff
@@ -0,0 +1,7 @@
+package com.singh.astha.notificationservice.enums;
+
+public enum DeviceType {
+
+    ANDROID,  IOS
```

Comment:
nit: Prefer to define enums on separate line and each followed by a trailing comma

Reply:
done